recipes/euporie/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "euporie" %}
+{% set version = "0.1.11" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/euporie-{{ version }}.tar.gz
+  sha256: 7d05980dbfcaa381e79b201ce05d7fe8e2bca62811f28e7b170211b0ba5c7bfc
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - euporie = euporie.app:App.launch
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+    - poetry
+  run:
+    - jsonschema >=3.2.0,<4.0.0
+    - jupyter_client ==6.1.12
+    - nbformat >=5.1.3,<6.0.0
+    - pillow  #>=8.2.0,<9.0.0
+    - prompt-toolkit >=3.0.18,<4.0.0
+    - python >=3.6,<4.0
+    - rich >=10.1.0,<11.0.0
+
+test:
+  imports:
+    - euporie
+    - euporie.libs.commonmark_extensions
+  commands:
+    # - pip check
```

Comment:
Can you add a comment why this us commented?

Reply:
Done - basically I am waiting for pillow 8.2.0 to be released as conda package https://github.com/conda-forge/pillow-feedstock/pull/93